src/components/SidebarMenu.vue

```diff
@@ -62,18 +62,20 @@
           @afterEnter="expandAfterEnter"
           @beforeLeave="expandBeforeLeave"
         >
-          <listItem
-            v-if="mobileItem && mobileItem.child"
-            :items="mobileItem.child"
-            :show-child="showChild"
-            :rtl="rtl"
-            :is-collapsed="isCollapsed"
-          >
-            <slot
-              slot="dropdown-icon"
-              name="dropdown-icon"
-            />
-          </listItem>
+          <div class="vsm--list">
```

Comment:
v-if needed to be in the vsm--list
```suggestion
          <div class="vsm--list" v-if="mobileItem && mobileItem.child">
```